### PR TITLE
Several improvements to superpmi.py

### PR DESCRIPTION
**Pull request by BruceForstall**
1. The "diff" JIT in asmdiffs is now assumed to be in the current
repo if not specified. (That is, a computed default is attempted.)
2. jit-analyze is run on the generated diffs, if it is found on the
PATH. This has some limitations, as we generate a file for each
function, so the "top file" sections are not interesting.
3. Elapsed time is computed and displayed at the end of the task.

### CODE REVIEW

**Comment by BruceForstall** [comment]
@dotnet/jit-contrib

File: src/coreclr/scripts/superpmi.py [hunk 1]
```diff
@@ -246,6 +244,75 @@ def safe_char(c):
...
+    Returns:
+        (str) The pathname of the object, or None if not found.
+    """
+    for dirname in pathlist:
+        candidate = os.path.join(dirname, name)
+        if matchFunc(candidate):
+            return candidate
+    return None
+
+def findFile(filename, pathlist):
```
Comment (sandreenko) [inline src/coreclr/scripts/superpmi.py]: could you please rename it to `find_file` while you are here? Looks like that file mostly uses snake_case.

File: src/coreclr/scripts/superpmi.py [hunk 2]
```diff
@@ -246,6 +244,75 @@ def safe_char(c):
...
+    Args:
+        filename (str)          : name to search for
+        pathlist (str)          : semicolon-separated string of directory names to search
+
+    Returns:
+        (str) The pathname of the object, or None if not found.
+    """
+    return _find(filename, pathlist)
+
+def findDir(dirname, pathlist):
```
Comment (sandreenko) [inline src/coreclr/scripts/superpmi.py]: the same here.

**Review by sandreenko — APPROVED** [review]
LGTM, thank you

**Pull request merged**